app/assets/javascripts/alchemy/alchemy.internal_links_select.js.coffee

```diff
@@ -0,0 +1,35 @@
+class window.Alchemy.InternalLinksSelect
+  constructor: (options) ->
+    @$frame = options.frame
+    @$selectField = options.selectField
+    @$onChange = options.onChange
+
+    @initInternalAnchors()
+
+  # Find Alchemy Elements presents on DOM and populate the select with them
+  selectElementsInternalIds: (elements) =>
+    if elements.elements.length > 0
+      $.each elements.elements, (_, element) =>
+        DOMElement = @$frame.contents().find("[data-alchemy-element=\"#{element.id}\"]")
```

Comment:
Uppercase variables should be avoided as they could be confused with "class" names. Also I prefer to use `$` prefixed variables if a jQuery object are stored into them